client/browser/src/shared/code-hosts/gerrit/codeHost.module.scss

```
@@ -0,0 +1,49 @@
+.hover-overlay--gerrit {
```

Let's remove code host suffixes from CSS modules.
```suggestion
.hover-overlay {
```